cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -497,6 +499,26 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         })) {
           statsOnlyResultTable()
         }
+        p("The ability to modify the name of users can be granted via the `SET USER NAME` privilege. The following query shows an example of this:")
+        query("GRANT SET USER NAME ON DBMS TO statusModifier", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        p("The resulting role should have privileges that only allow modifying the name of users:")
+        query("SHOW ROLE statusModifier PRIVILEGES", NoAssertions) {
+          p("Lists all privileges for role 'statusModifier'")
+          resultTable()
+        }
```

Comment:
I think these roles are one per privilege, so this one is for status and name should have its own